docs/src/test/java/jdbi/doc/CallTest.java

```diff
@@ -15,21 +17,24 @@
     public PostgresDbRule db = new PostgresDbRule();
 
     @Test
-    public void testCall()
-    {
+    public void testCall() {
         Handle handle = db.getSharedHandle();
-        // tag::call[]
-        handle.execute(
-                "CREATE FUNCTION the_answer(answer INOUT INTEGER) AS $$" +
-                    "BEGIN answer := 42; END;" +
-                "$$ LANGUAGE plpgsql");
-
-        assertThat(handle.createCall("{? = call the_answer(?)}")
-                .registerOutParameter(0, Types.INTEGER)
-                .bind(1, 13)
-                .invoke()
-                .getInt(0))
-            .isEqualTo(42);
-        // end::call[]
+
+        handle.execute(findSqlOnClasspath("create_stored_proc_add"));
+
+        // tag::invokeProcedure[]
+        OutParameters result = handle
+                .createCall("{:sum = add(:a, :b)}") // <1>
```

Comment:
this isn't working

Reply:
Fixed. Will merge when the Travis build completes.